content/en/tracing/profiler/enabling/linux.md

```diff
@@ -93,6 +91,27 @@ exec ./ddprof --environment prod --service my-web-app --service_version 1.0.3 my
 
 5. A minute or two after starting your application, your profiles appear on the [Datadog APM > Profiler page][5].
 
+## Library Installation
+
+If you would rather use a library instead of a standalone executable, the profiler is available as both shared and static libraries. There are a few notable similarities and differences between the library and standalone versions:
+
+* The profiling library is only available for Linux v4.17+ on `amd64` or `arm64`.
+* The profiling library has the same requirements (items 1 and 2 in the [Installation](#installation) section) as the standalone executable.
+* The behavior of the library is nearly identical to the standalone executable, including the sandboxing features that isolate the profiler from your application.
+* The profiling library interfaces check environment variables, but they do not have other means of configuration-passing.
+
+For generality, the library exposes a C API. Here is an example of incorporating the library into a C application.
```

Comment:
There is a different way to start up the application by using LD_PRELOAD
Nicolas has an auto-startup that is perhaps interesting to mention. This covers the Rust use case for instance. Do you think we should mention it ?

Reply:
Yes, I think that's a good point.  I'll mention it.